packages/kbn-text-based-editor/src/text_based_languages_editor.tsx

```diff
@@ -137,6 +166,7 @@ export const TextBasedLanguagesEditor = memo(function TextBasedLanguagesEditor({
   const [editorHeight, setEditorHeight] = useState(
     isCodeEditorExpanded ? EDITOR_INITIAL_HEIGHT_EXPANDED : EDITOR_INITIAL_HEIGHT
   );
+  const [hasSmallerSpace, setHasSmallerSpace] = useState(false);
```

Comment:
nit: nonblocker, but how about changing the name to one of these: `isSpaceReduced` or `isLayoutCompact`, `isAreaReduced`

Reply:
Good point dear, I will go with the isSpaceReduced I think

Reply:
Done https://github.com/elastic/kibana/pull/169911/commits/25b28b6156f93162edd38741a426c030477001ac